src/context/authContext.js

```diff
@@ -0,0 +1,51 @@
+import { createContext, useContext, useState } from 'react';
+import axios from 'axios';
+
+const AuthContext = createContext();
+
+export const useAuth = () => useContext(AuthContext);
+
+export const AuthProvider = ({ children }) => {
+
+    const [user, setUser] = useState(null);
+    const [encodedToken, setEncodedToken] = useState();
+
+    const SignupUser = async (email, password) => {
```

Comment:
While signup you have to send first name and last name inside async as well. You are sending only email and password

Reply:
Done in commit [5d44d50](https://github.com/shivanigangadharan/streambucket/pull/1/commits/5d44d50992faa27cf26e6bbf0724535aba5b7a15).